.github/workflows/generate-alpha-tag.yaml

```diff
@@ -0,0 +1,59 @@
+name: generate-alpha-tag
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.parallel=false -Dorg.gradle.jvmargs="-Xmx5g -XX:+UseParallelGC -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8"
```

Comment:
meta needs no daemon and no parallel to work too?

Reply:
as in our chat, we could take a look if we can improve this in another PR

Reply:
as in our chat, we could take a look if we can improve this in another PR